src/platform/forms-system/src/js/fields/AutosuggestField.jsx

```diff
@@ -208,6 +208,19 @@ export default class AutosuggestField extends React.Component {
     const { idSchema, formContext, formData, uiSchema, schema } = this.props;
     const id = idSchema.$id;
 
+    // wrap matching text in a <mark> element
+    const highlightText = uiSchema['ui:options']?.highlightText;
```

Comment:
```suggestion
    const highlightText = uiSchema['ui:options']?.highlightText ?? true;
```

This seems like a solid improvement, so I'd recommend enabling this by default. Fields can manually disable this as desired.